packages/web/src/components/ui/HeaderNew/HeaderNew.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styles from './HeaderNew.module.scss';
+import Container from '../Container/Container';
+import { Icon } from '../Icon/Icon';
+import { trackCustomEvent } from '../../../services/analytics';
+
+export default function HeaderNew() {
```

Comment:
Will it be renamed in future? I think we won't leave old components in the repo, so it could be just 'Header'

Reply:
Yes, it will. For now i'm using `New` suffix to keep old ones alongside as references, especially when it's not just static markup, but some functionality involved.

In case of `Header` it could easily be replaced, because it's mostly static links, so i'll do it in future commits/updates.

Reply:
Let's change existing components instead of adding a suffix